Framework/PythonInterface/mantid/utils/logging.py

```diff
@@ -0,0 +1,70 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+# local packages
+from mantid.kernel import ConfigService
+
+# standard packages
+from contextlib import contextmanager
+import os
+import sys
+import tempfile
+
+
+@contextmanager
+def to_file(level=None, filename=None):
```

Comment:
How is this different than setting the file channel?

Reply:
what file channel are you referring to?

Reply:
passing `FileChannel` to the configuration service is not understood, meaning the the `Poco` file channel is not set, hence the trick to redirect `sys.stdout` to write to a file